README.md

```diff
@@ -14,6 +14,75 @@ Thumbprint Tokens are published as JavaScript, SCSS, Swift, and Kotlin.
 
 You can use the [Thumbprint Tokens GraphQL server](https://thumbprint-tokens.netlify.com/) to query tokens and view the schema.
 
+Button, Theme, Title Color, Active, primary Tokens should follow this recommended hierarchy: `[<component-name>] <category-name> [<option-name> <state>] <token>`.
```

Comment:
I wonder how something like dark mode or themes would fit into this convention.

Currently we have:

```
Button-Theme-TitleColor-Active-primary
```

Would it be:

```
Button-Theme-TitleColor-Active-primary-dark
Button-Theme-TitleColor-dark-Active-primary
Button-Theme-TitleColor-ActiveDark-primary
```

Or something else? It might not matter in the short term, but it may worthwhile to consider this now since it'll be difficult to migrate if it requires a future breaking change.